css/components/topics/topics-thumbnail-list.scss

```diff
@@ -50,14 +52,14 @@
     &:hover {
       box-shadow: 0 20px 30px rgba($black, .2);
       transform: translateY(-4px);
+      transition: all .3s;
```

Comment:
We have custom animations and easings defined => `all $animation-time-2 $ease-in-out-sine;` If you are animating opacity or shadows use `linear`